msu/utils/math.nut

```diff
@@ -28,4 +28,15 @@
 		::MSU.requireFloat(_min, _max);
 		return _min + (::Math.rand(0, 2147483647) / 2147483647.0) * (_max - _min);
 	}
+
+	function roundSig( _num, _significantFigures )
+	{
+		::MSU.requireInt(_significantFigures);
+		if (_num == 0)
+			return 0;
+
+		local d = ceil(log10(_num < 0 ? -_num : _num));
```

Comment:
use `::Math.abs` instead

Reply:
I've use `fabs` which is a native function and hopefully faster/more efficient than `::Math.abs`.